src/components/fields/radio-button/types.ts

```diff
@@ -1,12 +1,13 @@
 import { RadioButtonProps } from "@lifesg/react-design-system/radio-button";
-import { TComponentOmitProps } from "../../frontend-engine";
+import { TComponentOmitProps, TFrontendEngineFieldSchema } from "../../frontend-engine";
 import { IBaseFieldSchema } from "../types";
 
 interface IOption {
 	label: string;
 	value: string;
 	disabled?: boolean | undefined;
 	imgSrc?: string | undefined;
+	children?: Record<string, TFrontendEngineFieldSchema>;
```

Comment:
similar changes as checkbox
- add undefined type
- need to differentiate option type between toggle styleType and others